ktorm-support-mysql/src/test/kotlin/me/liuwj/ktorm/support/mysql/MySqlTest.kt

```diff
@@ -232,4 +235,28 @@ class MySqlTest : BaseTest() {
 
         assert(name == "VINCE")
     }
+
+    @Test
+    fun testSumIf() {
+        val countRich = database.from(Employees)
+                .select(
+                        sum(IF((Employees.salary greaterEq 100L).cast(BooleanSqlType),
+                                then = ArgumentExpression(1, IntSqlType),
+                                otherwise = ArgumentExpression(0, IntSqlType)))
+                )
+                .map { row -> row.getInt(1) }
+        assert(countRich.size == 1)
+        assert(countRich.first() == 3)
+    }
+
+    @Test
+    fun testSum() {
+        val countRich = database.from(Employees)
+                .select(
+                        sum(Employees.salary.greaterEq(100L).cast(IntSqlType))
```

Comment:
I'll be good to have a `toInt()` function for `ColumnDeclaring<Boolean>`.

Reply:
Added